Updated steps to deploy with lifecycle manager (#121)
kyma-project/nats-manager: README.md

make uninstall
```

### Deploy NATS Manager module with [Kyma Lifecycle Manager](https://github.com/kyma-project/lifecycle-manager/tree/main)

1. Deploy the Lifecycle Manager to the Kubernetes cluster:

   ```shell
   kyma alpha deploy
   ```

2. Apply the NATS module template to the Kubernetes cluster:

>    > **NOTE:** You can get the latest released module template from [here](https://github.com/kyma-project/nats-manager/releases/latest/download/module-template.yaml), or you can use module template from the artifacts of `nats-module-build` job from `main` branch or from your pull request.

Reviewer: ```suggestion
   > **NOTE:** You can get the latest released [module template](https://github.com/kyma-project/nats-manager/releases/latest/download/module-template.yaml), or you can use the module template from the artifacts of `nats-module-build` job either from the `main` branch or from your pull request.
```

   ```sh
   kubectl apply -f module-template.yaml
   ```

3. Enable the NATS module:

   ```sh
   kyma alpha enable module nats -c fast -n kyma-system -p Ignore
   ```

> 4. [Optional] Install NATS Custom Resource:

Reviewer: ```suggestion
4. Optionally, install NATS Custom Resource:
```

    ```sh
    kubectl apply -f config/samples/eventing-nats-eval.yaml
    ```

> 5. [Optional] Check whether your NATS module is deployed properly:

Reviewer: ```suggestion
5. If you want to verify whether your NATS module is deployed properly, perform the following checks:
```
